net/libfab/libfab.c

```diff
@@ -92,12 +92,73 @@ M0_INTERNAL void m0_net_libfab_fini(void)
 	*/
 }
 
+/**
+ * Used to take the ip and port from the given end point
+ * ep_name : endpoint address from domain
+ * node    : copy ip address from ep_name
+ * port    : copy port number from ep_name
+ * Example of ep_name IPV4 192.168.0.1:4235
```

Comment:
Please note, the 'sock' xprt recognize the lnet ep format.
Does libfab need the same?
I think we it's better to use the lnet compatible format. So, a lot of code can keep the same.

Please discuss this with @nikitadanilov